pipelines/restart_grok46_33.sh

```
@@ -0,0 +1,41 @@
+
+ROOT=/home/raulmc/rmems/synthetic-factory
+PROMPT="$ROOT/pipelines/restart_grok46_33.md"
+GROK="${GROK:-/home/raulmc/.local/bin/grok}"
+LOG=/tmp/grok46-restart-33.log
+LOCK=/tmp/grok46-restart-33.lock
+SESSION=grok46-agentic-restart
+
+export TZ=America/Chicago
+now_hm=$(date +%H%M)
+# Weekly reset is 06:52 CDT. Allow from 06:52 through 08:30 so a delayed cron still fires.
+if [[ "$now_hm" < "0652" || "$now_hm" > "0830" ]]; then
```

Shellcheck (0.11.0)

[error] < is for string comparisons. Use -lt instead.

(SC2071)